packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap

```diff
@@ -2636,13 +2636,13 @@ exports[`list of all events 1`] = `
               color="light"
               font-size="3"
             />
-            App Session Started
```

Comment:
Confused as to why this would be removed given this change.

Reply:
It looks like `APP_SESSION_START` has the same event code:
https://github.com/gravitational/webapps/blob/723fc13fe0184a9e67bbb6893232f0a55c136254/packages/teleport/src/services/audit/types.ts#L49